src/fparser/two/utils.py

```diff
@@ -335,6 +364,8 @@ def __new__(cls, string, parent_cls=None):
             obj = object.__new__(cls)
             obj.string = string
             obj.item = None
+            # Set-up parent information for the results of the match
+            _set_parent(obj, result)
```

Comment:
As discussed, I think we should also have a list called `children` that is a member of `Base` and I think the simplest place to set it up would be here (perhaps _set_parent could become `_set_links` and do both?

Reply:
I've done this as a `children` property that transparently returns either `items` or `content`. Otherwise we would end up manually keeping a copy of either of those that we would have to keep up-to-date when the tree is manipulated.